packages/paste-website/src/components/site-wrapper/SiteBody.tsx

```diff
@@ -19,3 +33,40 @@ export const SiteBody = styled.div`
     grid-template-columns: ${SIDEBAR_WIDTH} 1fr;
   }
 `;
+
+export const SiteBody: React.FC = ({children}) => {
+  const {theme: activeTheme} = useActiveSiteTheme();
+  return (
+    <>
+      {activeTheme === 'default' && (
+        <Box
+          backgroundColor="colorBackgroundWarningLightest"
+          borderRadius="borderRadius20"
+          padding="space20"
+          paddingLeft="space40"
+          paddingRight="space40"
+        >
+          <Text as="p" textAlign="center">
+            <strong>WARNING:</strong> The Paste theme is an <em>extremely early</em> preview of future work!{' '}
+            <Anchor
+              href="https://docs.google.com/document/d/1H2Rj3NEmVSv0yxMBRjYOjruQllO__uj2-I_ibIoumZs/edit?usp=sharing"
+              target="_blank"
+            >
+              Read the FAQ
+            </Anchor>{' '}
```

Comment:
Just an FYI, Scott is working on an FAQ page for the Paste doc site (https://share.goabstract.com/0ee0c320-f4f8-483c-a56e-87fb301890c3?sha=59a451d9a07b0cd0162e4566dd63e244d1ff1838), and I've already asked him to mock it up with the unified design language FAQs. When we build this page, we can swap this link over 💪

Reply:
@serifluous amazing, thank you so much for doing that!